src/Theme/Theme.js

```diff
@@ -0,0 +1,112 @@
+import {StyleSheet} from 'react-native';
+
+export const getStyle = (stylekey, theme) => {
+  const styles = StyleSheet.create({
+    container: {
+      backgroundColor: theme === LIGHT_MODE ? '#E5E5E5' : '#1F232C',
+      paddingHorizontal: 35,
+      paddingVertical: 23,
+      height: '100%',
+    },
+    header: {
+      flexDirection: 'row',
+      paddingBottom: 56,
+    },
+    heading: {
+      paddingBottom: 15,
+    },
+    list: {
+      backgroundColor: getFillColor(theme),
+      borderWidth: 1,
+      borderColor: theme === LIGHT_MODE ? '#FFFF' : '#111827',
+      maxHeight: 370,
+      marginBottom: 15,
+    },
+    toolButtonContainer: {
+      fontFamily: 'Inter',
+      fontStyle: 'normal',
+      fontWeight: '600',
+      fontSize: 16,
+      color: getFillColor(theme),
+      width: 'auto',
+      alignSelf: 'center',
+      borderRadius: 100,
+      backgroundColor: '#06B6D4',
+      alignItems: 'center',
+    },
+    toolButton: {
+      margin: 13,
+    },
+    sectionHeadingText: {
+      fontFamily: 'Inter',
+      fontStyle: 'normal',
+      fontWeight: '600',
+      fontSize: 18,
+      lineHeight: 22,
+      color: theme === LIGHT_MODE ? '#111827' : '#FFFF',
+      paddingBottom: 15,
+    },
+    listItemText: {
+      fontFamily: 'Inter',
+      fontStyle: 'normal',
+      fontWeight: 'normal',
+      fontSize: 16,
+      lineHeight: 19,
+      color: getTextColor(theme),
+      maxWidth: 320,
+    },
+    listSeparator: {
+      flex: 1,
+      height: 1,
+      backgroundColor: theme === LIGHT_MODE ? '#E5E5E5' : '#1F232C',
+    },
+    categoryTextStyle: {
+      fontFamily: 'Inter',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      fontSize: 18,
+      color: theme === LIGHT_MODE ? '#E5E5E5' : '#FFFF',
+    },
+    textarea: {
+      fontFamily: 'Inter',
+      color: theme === LIGHT_MODE ? '#111827' : '#FFFF',
+      maxWidth: 360,
+      marginHorizontal: 'auto',
+      fontSize: 20,
+      lineHeight: 24.3,
+    },
+    input: {
+      color: '#111827',
+      lineHeight: 24.3,
+    },
+    textTextareaContainer: {
+      height: 100,
+      marginVertical: 0,
+      marginHorizontal: 'auto',
+      backgroundColor: theme === LIGHT_MODE ? '#FFF' : '#111827',
+      justifyContent: 'center',
+      alignItems: 'center',
+      borderStyle: 'solid',
+      borderColor: '#E5E5E5',
+    },
+  });
+  return styles[stylekey];
+};
+
+export const getFillColor = theme =>
+  theme === LIGHT_MODE ? '#FFFF' : '#111827';
+
+export const getTextColor = theme =>
+  theme === LIGHT_MODE ? '#111827' : '#FFFF';
+
+export const getFillLearningColor = theme =>
+  theme === LIGHT_MODE ? '#FFFF' : '#111827';
+
+export const DARK_MODE = 'dark';
+
+export const LIGHT_MODE = 'light';
+
```

Comment:
Please store all specific keys for component styles in consts to avoid typos and to improve reusability

`const export LIST_STYLE = 'list'`